powertools-batch/src/main/java/software/amazon/lambda/powertools/batch/BatchMessageHandlerBuilder.java

```diff
@@ -0,0 +1,38 @@
+package software.amazon.lambda.powertools.batch;
+
+import software.amazon.lambda.powertools.batch.builder.*;
+
+/**
+ * A builder-style interface we can use within an existing Lambda RequestHandler to
+ * deal with our batch responses. A second tier of builders is returned per-event-source
```

Comment:
> A second tier of builders is returned per-event-source...

excuse my english level but I don't understand this comment

Reply:
No worries at all; that's a shocking sentence. I'll rewrite it.